modules/configuring-firewall.adoc

```diff
@@ -46,14 +46,14 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com`
 
-|`rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com`
+|`rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com`, +
+`rhcos.mirror.openshift.com`
```

Comment:
We may have this in the wrong place since these redirectors are not a container registry. I think it belongs in the table down at roughly line 127.

Reply:
@mike-nguyen is this the case for #48037 as well?

Reply:
Indeed, it looks like we originally added to the wrong table.

Reply:
I see, so the BZ ticket is incorrect about the location? 

Which URL should be replaced in the table around line 127? Is it the `art-rhcos-ci.s3.amazonaws.com` URL?

Reply:
`rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com` and `rhcos.mirror.openshift.com`
should replace `art-rhcos-ci.s3.amazonaws`.   `art-rhcos-ci.s3.amazonaws` is a direct link to our s3 bucket which does not seem to be used in the installer code anymore (at least for 4.10).  I am not aware of it being used anywhere else.

Reply:
gotcha, thanks both! I'll update this PR after I make a new 4.11 PR with the update